packages/wallet/src/W3mFrameSchema.ts

```diff
@@ -41,7 +41,8 @@ export const AppUpdateEmailPrimaryOtpRequest = z.object({ otp: z.string() })
 export const AppUpdateEmailSecondaryOtpRequest = z.object({ otp: z.string() })
 export const AppSyncThemeRequest = z.object({
   themeMode: z.optional(z.enum(['light', 'dark'])),
-  themeVariables: z.optional(z.record(z.string(), z.string().or(z.number())))
+  themeVariables: z.optional(z.record(z.string(), z.string().or(z.number()))),
+  w3mThemeVariables: z.record(z.string(), z.string())
```

Comment:
if this is not `optional` it will require installing a new v on secure-site to prevent zod crashes

Reply:
Let's go with the new version way. This should be required when we call `syncTheme` function to keep our themes synced, wdyt? @tomiir